.circleci/config.yml

```diff
@@ -434,7 +439,7 @@ workflows:
 
       - test-main:
           name: test-core-node
-          test-params: -k node src packages/micropip packages/fpcast-test packages/sharedlib-test-py/ packages/cpp-exceptions-test/ pyodide-test-runner
+          test-params: -k node src packages/micropip packages/fpcast-test packages/sharedlib-test-py/ packages/cpp-exceptions-test/ pyodide-test-runner/pyodide_test_runner/tests
```

Comment:
Yes, that sounds reasonable, thanks!

I updated the test path to `pyodide-test-runner/pyodide_test_runner`. (`pytest` complains when I set the test path to `pyodide-test-runner` because in that case `pyodide_test_runner` import path is ambiguous)

Reply:
> I updated the test path to `pyodide-test-runner/pyodide_test_runner`. (`pytest` complains when I set the test path to `pyodide-test-runner` because in that case `pyodide_test_runner` import path is ambiguous)

Oh, it's failing... perhaps I should write separate test job for pyodide-test-runner.